### PR TITLE
Expand `Guest difficulty`

### PR DESCRIPTION
**Pull request by Nivalyx**
tried to expand this stub article with more bits of information, hopefully everything's good to go tho whew

---

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Beatmap/Guest_difficulty/en.md [hunk 1]
```diff
@@ -11,6 +10,16 @@ tags:
...
 
-Many guest difficulties are often requested by private messages between creators. Although GDs can also be requested in [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
+Although not mandatory, having guest difficulties can be beneficial in many occasions as they often feature a completely different mapping approach between one another (therefore diversifying the mapset) and help reduce mapping burnout. Guest difficulties are most commonly requested through private messages between creators, although they may also be requested by other means such as the in-game `#mapping` channel or [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
+
+Guest difficulties are not to be confused with difficulties that feature multiple creators, which are instead known as [collaborative difficulties](/wiki/Beatmap/Beatmap_collaborations).
+
+## Guest difficulty ownership
+
+![](img/gd_ownership.png)
```
Comment (ZeroPyrozen) [inline wiki/Beatmap/Guest_difficulty/en.md:21]:
```suggestion
::: Infobox
![](img/gd_ownership.png "An example of guest difficulty ownership")
:::
```

File: wiki/Beatmap/Guest_difficulty/en.md [hunk 2]
```diff
@@ -11,6 +10,16 @@ tags:
...
 
-Many guest difficulties are often requested by private messages between creators. Although GDs can also be requested in [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
+Although not mandatory, having guest difficulties can be beneficial in many occasions as they often feature a completely different mapping approach between one another (therefore diversifying the mapset) and help reduce mapping burnout. Guest difficulties are most commonly requested through private messages between creators, although they may also be requested by other means such as the in-game `#mapping` channel or [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
+
+Guest difficulties are not to be confused with difficulties that feature multiple creators, which are instead known as [collaborative difficulties](/wiki/Beatmap/Beatmap_collaborations).
+
+## Guest difficulty ownership
```
Comment (cl8n) [inline wiki/Beatmap/Guest_difficulty/en.md:19]:
```suggestion
## Ownership
```
...maybe. just seems redundant.

File: wiki/Beatmap/Guest_difficulty/en.md (hunk 1 above)
Comment (cl8n) [inline wiki/Beatmap/Guest_difficulty/en.md:21]: could use a simple title

```suggestion
![](img/gd_ownership.png "\"Difficulty Owner\" menu on the website")
```

File: wiki/Beatmap/Guest_difficulty/en.md [hunk 3]
```diff
@@ -11,6 +10,16 @@ tags:
 
 *For regulations surrounding guest difficulties, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
 
-A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username. A beatmap featuring multiple creators is also known as a [collaborative difficulty](/wiki/Beatmap/Beatmap_collaborations).
+A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name which contains the guest creator's username.
```
Comment (Walavouchey) [inline wiki/Beatmap/Guest_difficulty/en.md:13]:
```suggestion
A **guest difficulty** (*GD*) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name which contains the guest creator's username.
```

File: wiki/Beatmap/Guest_difficulty/en.md [hunk 4]
```diff
@@ -11,6 +10,16 @@ tags:
...
 
-Many guest difficulties are often requested by private messages between creators. Although GDs can also be requested in [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
+Although not mandatory, having guest difficulties can be beneficial in many occasions as they often feature a completely different mapping approach between one another (therefore diversifying the mapset) and help reduce mapping burnout. Guest difficulties are most commonly requested through private messages between creators, although they may also be requested by other means such as the in-game `#mapping` channel or [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
+
+Guest difficulties are not to be confused with difficulties that feature multiple creators, which are instead known as [collaborative difficulties](/wiki/Beatmap/Beatmap_collaborations).
```
Comment (Walavouchey) [inline wiki/Beatmap/Guest_difficulty/en.md:17]:
```suggestion
Guest difficulties are not to be confused with difficulties that feature multiple creators, which are instead known as *[collabs](/wiki/Beatmap/Beatmap_collaborations)*.
```

File: wiki/Beatmap/Guest_difficulty/en.md (hunk 3 above)
Comment (cl8n) [inline wiki/Beatmap/Guest_difficulty/en.md:13]: very nitpicky/technical but "containing" isn't the same as "which contains" here and I think the verb was intentional. the former makes "difficulty name containing the guest creator's username" the way to identify GDs, while the latter reduces it to just "difficulty name", followed by explanation of what's in the diff name. the latter also makes a definite statement abt what to expect in the diff name, even though this sentence begins with "can *usually* be identified"

so basically...
```suggestion
A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username.
```

File: wiki/Beatmap/Guest_difficulty/en.md [hunk 5]
```diff
@@ -11,6 +10,16 @@ tags:
...
+
+## Guest difficulty ownership
+
+![](img/gd_ownership.png)
+
+On the [beatmap discussion page](wiki/Beatmap_Discussion), the beatmap host may hand over ownership of guest difficulties present in the beatmap to their respective creators through the `Difficulty Owner` button. This allows for the guest difficulty owners to answer and resolve pending issues in their difficulties on their own.
```
Comment (cl8n) [inline wiki/Beatmap/Guest_difficulty/en.md:23]:
```suggestion
On the [beatmap discussion page](wiki/Beatmap_Discussion), the beatmap host may transfer ownership of guest difficulties to their respective creators through the `Difficulty Owner` button. This allows for the guest difficulty owners to resolve pending issues in their difficulties on their own.
```

**Review by cl8n — DISMISSED** [review]
good unstub :tada: 

probably do the `outdated_since` stuff. and rebase with better commit messages

File: wiki/Beatmap/Guest_difficulty/en.md [hunk 6]
```diff
@@ -11,6 +10,16 @@ tags:
...
+A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name which contains the guest creator's username.
 
-Many guest difficulties are often requested by private messages between creators. Although GDs can also be requested in [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
+Although not mandatory, having guest difficulties can be beneficial in many occasions as they often feature a completely different mapping approach between one another (therefore diversifying the mapset) and help reduce mapping burnout. Guest difficulties are most commonly requested through private messages between creators, although they may also be requested by other means such as the in-game `#mapping` channel or [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
```
Comment (Walavouchey) [inline wiki/Beatmap/Guest_difficulty/en.md:15]:
```suggestion
Although not mandatory, having guest difficulties can be beneficial in many occasions as they often feature a completely different mapping style between one another, which diversifies the mapset, and help reduce mapping burnout. Guest difficulties are most commonly requested through private messages between creators, although they may also be requested by other means such as the in-game `#mapping` channel or [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
```

**Review by Walavouchey — COMMENTED** [review]
seems good

File: wiki/Beatmap/Guest_difficulty/en.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Beatmap/Guest_difficulty/en.md:15]: I think you're reading into it a bit far - all it's saying is that GDs make sets more varied and reduce the load on the host, and I'd say that's a very mild take at best

File: wiki/Beatmap/Guest_difficulty/en.md (hunk 3 above)
Comment (Nivalyx) [reply wiki/Beatmap/Guest_difficulty/en.md]: o i think i'll leave this as is since i feel like it's rather awkward to have that description omitted from the line

File: wiki/Beatmap/Guest_difficulty/en.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Nivalyx) [reply wiki/Beatmap/Guest_difficulty/en.md:15]: decided to go with walavouchy's suggestion instead http://github.com/ppy/osu-wiki/pull/6482#discussion_r760791046

**Comment by Nivalyx** [comment]
ok i think that's all, this pr should be good to go now @Walavouchey @cl8n 👀 

i dunno where to get the hash for the `outdated_since` tag tho, so yeah i'll be sticking with the old `outdated: true` i guess 😔

**Comment by ZeroPyrozen** [comment]
you can get the `outdated_since` value from first commit SHA  of your `en.md` edit

File: wiki/Beatmap/Guest_difficulty/en.md [hunk 7]
```diff
@@ -11,6 +10,18 @@ tags:
 
 *For regulations surrounding guest difficulties, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
 
-A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username. A beatmap featuring multiple creators is also known as a [collaborative difficulty](/wiki/Beatmap/Beatmap_collaborations).
+A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username.
```
Comment (Walavouchey) [inline wiki/Beatmap/Guest_difficulty/en.md:13]: > o i think i'll leave this as is since i feel like it's rather awkward to have that description omitted from the line

i suggested removing it to be consistent with other instances of acronyms, but i suppose something like this could work as well. i disagree with having parentheses if it fits fit the flow of the sentence anyway (most of the time)

so either what i suggested or this

```suggestion
A **guest difficulty*, or *GD* for short, is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Beatmap/Guest_difficulty/en.md (hunk 7 above)
Comment (peppy) [reply wiki/Beatmap/Guest_difficulty/en.md:13]: I think your suggestion is missing a `*`.

**Comment by Nivalyx** [comment]
@Walavouchey yep that'll do 👀

File: wiki/Beatmap/Guest_difficulty/en.md [hunk 8]
```diff
@@ -11,6 +10,18 @@ tags:
 
 *For regulations surrounding guest difficulties, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
 
-A **guest difficulty** (or *GD* for short) is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username. A beatmap featuring multiple creators is also known as a [collaborative difficulty](/wiki/Beatmap/Beatmap_collaborations).
+A **guest difficulty**, or *GD* for short, is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). They can usually be identified by their difficulty name containing the guest creator's username.
```
Comment (TicClick) [inline wiki/Beatmap/Guest_difficulty/en.md:13]:
```suggestion
A **guest difficulty**, or *GD* for short, is a [difficulty](/wiki/Beatmap/Difficulty) of a [beatmap](/wiki/Beatmap) that is not created by the [beatmap host](/wiki/Beatmap/Beatmap_host). It can usually be identified by its name containing the guest mapper's username.
```

File: wiki/Beatmap/Guest_difficulty/en.md [hunk 9]
```diff
@@ -11,6 +10,18 @@ tags:
...
 
-Many guest difficulties are often requested by private messages between creators. Although GDs can also be requested in [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
+Although not mandatory, having guest difficulties can be beneficial in many occasions as they often feature completely different mapping styles between one another, which diversifies the mapset, and help reduce mapping burnout. Guest difficulties are most commonly requested through private messages between creators, although they may also be requested by other means such as the in-game `#mapping` channel or [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
```
Comment (TicClick) [inline wiki/Beatmap/Guest_difficulty/en.md:15]:
```suggestion
Although not mandatory, having guest difficulties can be beneficial in many occasions as they often feature completely different mapping styles between one another, which diversifies the mapset, and helps reduce mapping burnout. Guest difficulties are most commonly requested through private messages between creators, although they may also be requested by other means such as the in-game `#mapping` channel or [modding queues](https://osu.ppy.sh/community/forums/60)<!-- TODO: should link to an article about queues, not the forum -->.
```

**Review by TicClick — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]
